docs/guides/ibm-circuit-function.ipynb

```diff
@@ -342,7 +342,11 @@
    "cell_type": "code",
    "execution_count": 8,
    "id": "4070e592",
-   "metadata": {},
+   "metadata": {
+     "tags": [
+       "raises-exception"
+     ]
```

Comment:
Nice feature of most Jupyter execution tools. The exception in this cell is intentional.